kythe/go/util/schema/schema.go

```diff
@@ -96,3 +97,35 @@ func (n *Node) ToEntries() []*spb.Entry {
 	}
 	return entries
 }
+
+// GetNodeKind returns the string representation of the node's kind.
+func GetNodeKind(n *scpb.Node) string {
+	if k := n.GetGenericKind(); k != "" {
+		return k
+	}
+	return NodeKindString(n.GetKytheKind())
+}
+
+// GetSubkind returns the string representation of the node's subkind.
+func GetSubkind(n *scpb.Node) string {
+	if k := n.GetGenericSubkind(); k != "" {
+		return k
+	}
+	return SubkindString(n.GetKytheSubkind())
+}
+
+// GetFactName returns the string representation of the fact's name.
+func GetFactName(f *scpb.Fact) string {
```

Comment:
Why the switch to Get* for the function names?

Reply:
The unprefixed names are already used.  :-/